extensions/info/deployment/src/main/java/io/quarkus/info/deployment/InfoBuildTimeConfig.java

```diff
@@ -0,0 +1,88 @@
+package io.quarkus.info.deployment;
+
+import java.util.Map;
+
+import io.quarkus.runtime.annotations.ConfigPhase;
+import io.quarkus.runtime.annotations.ConfigRoot;
+import io.smallrye.config.ConfigMapping;
+import io.smallrye.config.WithDefault;
+import io.smallrye.config.WithParentName;
+
+@ConfigMapping(prefix = "quarkus.info")
+@ConfigRoot(phase = ConfigPhase.BUILD_TIME)
+public interface InfoBuildTimeConfig {
+
+    /**
+     * Whether the info endpoint will be enabled
+     */
+    @WithDefault("true")
+    boolean enabled();
```

Comment:
My first thought was that we shouldn't expose it by default for security reasons but... the extension is not added by default, right?
So it's only exposed if the user adds the extension? If so, I'm good with it!

Reply:
Exactly, one would need the extension to get anything under `/q/info` in the first place.